pre-docs/bolt_installing_modules.md

```diff
@@ -1,6 +1,36 @@
 # Installing modules
 
-Set up Bolt to download and install modules.
+Bolt is packaged with a collection of useful modules intended to support common work flows. The included modules are described in the [Puppetfile](../Puppetfile) included in this repository. Modules and supporting documentation are publicly available on the [Puppet Forge](https://forge.puppet.com/).
+
+**NOTE**: Bundled content is NOT PROVIDED when bolt is installed via a ruby gem. It is recommended that bolt is [installed](./bolt_installing.md) as a package for most use cases.
+
+Modules with useful task and plan content:
+
+- [package](https://forge.puppet.com/puppetlabs/package): Install, uninstall, update, and check the status of packages.
+- [service](https://forge.puppet.com/puppetlabs/service): Manage and inspect the state of services. 
+- [puppet_conf](https://forge.puppet.com/puppetlabs/puppet_conf): Inspect and change the configuration options in the `puppet.conf` file.
+- [facts](https://forge.puppet.com/puppetlabs/facts): Retrieve facts from specified nodes.
+- [puppet_agent](https://forge.puppet.com/puppetlabs/puppet_agent): Install Puppet Agent package.
+
+Core types and providers for applying manifest code in plans: 
+
+- [augeas_core](https://forge.puppet.com/puppetlabs/augeas_core): Manage configuration files using Augeas.
+- [host_core](https://forge.puppet.com/puppetlabs/host_core): Manage host entries in a hosts file.
+- [scheduled_task](https://forge.puppet.com/puppetlabs/scheduled_task): Provider capable of using the more modern Version 2 Windows API for task management.
+- [sshkeys_core](https://forge.puppet.com/puppetlabs/sshkeys_core): Manage `SSH` `authorized_keys`, and `ssh_known_hosts` files. 
+- [zfs_core](https://forge.puppet.com/puppetlabs/zfs_core): Manage `zfs` and `zpool` resources.
+- [cron_core](https://forge.puppet.com/puppetlabs/cron_core): Install and manage `cron` resources.
+- [mount_core](https://forge.puppet.com/puppetlabs/mount_core): Manage mounted filesystems and mount tables.
+- [selinux_core](https://forge.puppet.com/puppetlabs/selinux_core): Manage Security-Enhanced Linux.
+- [yumrepo_core](https://forge.puppet.com/puppetlabs/yumrepo_core): Manage client yum repo configurations by parsing INI configuration files.
+- [zone_core](https://forge.puppet.com/puppetlabs/zone_core): Manage Solaris zone resources.
+
+Note that the `canary`, `aggregate` and `puppetdb_fact` modules are listed in the `Puppetfile` but are specifically implemented for bolt and not available on the forge.
```

Comment:
I think skip mentioning the Puppetfile here and provide a link to the directory that contains them